bi-connectors/PowerBIConnector/SqlOdbcPBIConnector.pq

```diff
@@ -186,10 +218,10 @@ SqlOdbcPBIConnector = [
     // PBIDS Handler
     DSRHandlers = [
       #"sqlodbc" = [
-        GetDSR = (server, schema, object, optional options) => [ protocol = "sqlodbc", address = [ server = server ] ],
-        GetFormula = (dsr, optional options) => () => 
+        GetDSR = (server, schema, object, optional options) => [ protocol = "sqlodbc", address = [ server = server, port = schema, useSSL = object ] ],
```

Comment:
Why does port = schema?

Reply:
I discovered that `port` and `useSSL` are passed as the second and third arguments.
I'm not sure whether we can rename function arguments, so I kept it as is. I agree it is weird.

Reply:
I think you can, have you tried to see if it still works?